### ISSUE TITLE
Using mpc for spotify playlists

### ISSUE THREAD

**Issue by fmarzocca** [mopidy/mopidy-spotify#226]
I am trying to use mpc command from a bash script to start playing some Spotify playlist, with no success.

If I add a Spotify album, it is working:
```
mpc add spotify:album:4KLATmGtZBcl8uliNUFp18
mpc play
```

but when I try to use a playlist I get errors:
```
mpc add spotify:user:spotify:playlist:3ZgmfR6lsnCwdffZUan8EA
error adding spotify:user:spotify:playlist:3ZgmfR6lsnCwdffZUan8EA: directory or file not found

```

**Comment by kingosticks** [comment]
I'm on Summer holidays right now but have you had a chance to read the project README? In particular the bit which talks about the status of playlist support?

**Comment by fmarzocca** [comment]
Hi Nick, thanks for the answer. The README file brings me to several tickets, but Ic ouldn't clearly understand at which development point they are.
Never mind, have a nice and happy holiday. I can wait for your return.

**Comment by fmarzocca** [comment]
Hi, I tried to understand if this is an issue, a "work in progress" or a "not applicable" problem, but I got confused. Is it possible to clarify?

**Comment by kingosticks** [comment]
This was fixed in f52babd10b1e187076e73f6cdbd2aaf56c7bbb60 and will be Mopidy-Spotify v4.0
